test/integration/short/client-execute-tests.js

```diff
@@ -1072,7 +1072,8 @@ describe("Client @SERVER_API", function () {
                 ),
             );
         });
-        vdescribe("dse-5.1.6", "with noCompact", function () {
+        // Potentially delete as it requires datastax DB?
+        /* vdescribe("dse-5.1.6", "with noCompact", function () {
             before(function (done) {
                 // While DSE 6.0 supports the NO_COMPACT option, there is no way to create
                 // COMPACT STORAGE tables other than creating with an older C* version and
```

Comment:
NO_COMPACT is not supported is ScyllaDB and it's considered to be very ancient, useless feature by us. Let's drop this test.